Tests/TestCodeGenConfigurations/EmbeddedInTarget-RelativeAbsolute/PackageTwo/Package.swift

```diff
@@ -22,15 +22,15 @@ let package = Package(
       name: "PackageTwo",
       dependencies: [
         .product(name: "ApolloAPI", package: "apollo-ios"),
+      ],
+      swiftSettings: [
+        .unsafeFlags(["-warnings-as-errors"])
```

Comment:
Needed to be moved into the shared package build settings, not the test target build settings.